src/ripple/shamap/impl/SHAMapSync.cpp

```diff
@@ -419,7 +419,7 @@ SHAMapAddNode SHAMap::addRootNode (Blob const& rootNode,
 
     assert (seq_ >= 1);
     auto node = SHAMapAbstractNode::make(
-        rootNode, 0, format, uZero, false, f_.journal ());
+        rootNode, 0, format, SHAMapHash{uZero}, false, f_.journal ());
```

Comment:
I took a closer look at this.  I'd rather not make this change now.  Instead I want to create a new P/R which `constexpr`izes `base_uint`, and then this would become:
````
constexpr SHAMapHash uZero{};
````
And at that time we can remove Vinnie's comment about cleaning this up.